### ISSUE TITLE
`Error opening script file:` when using full path on Cygwin

### ISSUE THREAD

**Issue by jkutner** [jkutner/mjruby#15]
From inside emacs on cygwin, running a script called script.rb.  Outside of emacs, calling 'jruby script.rb' it works.

```
====

-*- mode: ruby-compilation; default-directory: "~/workspace/test/" -*-
RubyComp started at Fri Sep 11 13:19:22

jruby /home/user/workspace/test/test_vector.rb
warning: --1.9 ignored
Error opening script file: /home/user/workspace/test/test_vector.rb (No such file or directory)

RubyComp exited abnormally with code 1 at Fri Sep 11 13:19:22

====
```

**Comment by jkutner** [comment]
Even more simple:

```
IEUser@IE9Win7 ~
$ jruby test.rb
hello world from vim

IEUser@IE9Win7 ~
$ jruby /home/IEUser/test.rb
Error opening script file: /home/IEUser/test.rb (No such file or directory)

```

**Comment by jkutner** [comment]
This works:

```
IEUser@IE9Win7 ~
$ jruby $(cygpath -w -p /home/IEUser/test.rb)
hello world 
```

So I need a way to run that command from mjruby. The [existing Bash script](https://github.com/jruby/jruby/blob/9.0.4.0/bin/jruby.bash) does this for classpaths and other things:

**Comment by jkutner** [comment]
Fixed by e25d45ce8377e3136190ef491437b5e53c767dad
